src/plugins/replace-tokens.js

```diff
@@ -58,7 +60,9 @@ function remarkTokenReplacer(options) {
       (token) => node.url && node.url.includes(token)
     );
 
-    if (!hasToken) return;
+    if (!hasToken) {
+        return
+    };
```

Comment:
Moving the semicolon here is a bit suspect.

Reply:
I suspect it has to do with the Semicolon marker. We should modify this recipe to make it aware of that.